finetuner/tuner/dataset/samplers.py

```diff
@@ -150,6 +151,7 @@ def _prepare_batches(self):
             batches.append(batch)
 
         self._batches = batches
+        random.shuffle(self._batches)
```

Comment:
there should be some option right? sometimes to debug is nice to disable?

Reply:
I don't think so. There will be no advantage of having shuffling disabled